### PR TITLE
Add the initial working page for Socials

### PR DESCRIPTION
**Pull request by prabhat1081**
Add the initial draft for the socials page (#27) inspired by [ICLR's](https://github.com/ICLR/iclr.github.io/blob/master/templates/pages/socials.html) page. 
Currently, the schedule is directly hard-coded in the socials.yml file, need to update once the format of input data is finalized.

social_sample

### CODE REVIEW

File: templates/components.html [hunk 1]
```diff
@@ -243,3 +243,52 @@ <h1>{{ sponsor_level.level }}</h1>
...
+  <div class="cards row">
+    {% for social in socials %}
+    <div class="col-md-6 col-sm-12 p-3"
+        id='{{social.name|replace(" ", "_")|replace("?", "")}}'
+        style="box-sizing: border-box;">
+        <div class="card"
+            style="display:block; overflow:hidden; width:100%;">
+            <div class="card-header text-center"
+                style="min-height: 400px; width:100%;">
+  <!--                    <a class="text-muted" href="#">-->
```
Comment (hao-fang) [inline templates/components.html:257]: remove this if not needed?

**Comment by prabhat1081** [comment]
Corresponding PR for sitedata_acl2020: https://github.com/acl-org/acl-2020-virtual-conference-sitedata/pull/16

File: templates/components.html (hunk 1 above)
Comment (prabhat1081) [reply templates/components.html]: Removed. Thanks

**Comment by dreasysnail** [comment]
Thank you both!

**Pull request merged**